pkg/questionModel/multiSelectList.go

```diff
@@ -0,0 +1,126 @@
+package questionModel
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/list"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+type MultiSelectList struct {
+	list     list.Model
+	selected map[int]item // which to-do items are selected
+	itemMap  map[item]string
+	header   string
+	question string
+	err      error
+}
+
+func (m *MultiSelectList) InitializeModel(data *BubbleTeaData) {
+	header, items, itemMap := createItems(data)
+	items = append(items, item("\n[ Submit ]"))
+
+	itemDelegate := itemDelegate{
+		renderUnselected: func(s string, index int) string {
+			if index == len(items)-1 {
+				return xLargeLeftPadding.Copy().Inherit(blurred).Render(s)
+			}
+			return mediumLeftPadding.Render(fmt.Sprintf("%s %s", m.getCheckBox(index), s))
+		},
+		renderSelected: func(s string, index int) string {
+			if index == len(items)-1 {
+				return xLargeLeftPadding.Copy().Inherit(focused).Render(s)
+			}
+			return focusTableItem(fmt.Sprintf("> %s %s", m.getCheckBox(index), s))
+		},
+	}
+
+	m.list = createModelList(items, itemDelegate, 0)
+	m.header = header
+	m.itemMap = itemMap
+	m.question = data.QuestionString
+
+	m.selected = make(map[int]item)
+	defaultIndexes := getDefaultOptionIndexes(data)
+	for _, defaultIndex := range defaultIndexes {
+		m.selected[defaultIndex] = items[defaultIndex].(item)
+	}
+}
+
+func (m *MultiSelectList) Init() tea.Cmd {
+	return nil
+}
+
+func (m *MultiSelectList) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	switch msg := msg.(type) {
+	case tea.WindowSizeMsg:
+		m.list.SetWidth(msg.Width)
+		return m, nil
+
+	case tea.KeyMsg:
+		switch msg.Type {
+		case tea.KeyCtrlC:
+			m.err = errors.New("User has quit before finishing question!")
```

Comment:
Does this error message print to the console? If so, I'd prefer something friendlier or matter-of-fact, or something that describes what will happen next (e.g., "Goodbye!" or "Exiting the questionnaire"). Hitting Ctrl+C is not improper behavior for a user, so I don't want the message to sound accusatory.